Add tutorial for ROS 2 galactic [12701] (#126)
eProsima/Fast-DDS-monitor: docs/rst/ros/ros.rst

.. _ros_section:

###########################
Fast DDS Monitor with ROS 2
###########################

> Fast DDS Monitor is a very useful tool to monitor and study a ROS 2 network.

Reviewer: ```suggestion
Fast DDS Monitor is a useful tool for monitoring and studying a ROS 2 network.
```

> The automatic discovery of entities in a local network allows to easily see the different Participants that

Author: Use capital letter for Participants to keep the documentation format

> are running, as its endpoints, the topics that each one is using, and even the network interfaces that are being

Reviewer: ```suggestion
are running, as its endpoints, the topics that each one is using, and even the network interfaces they are using
```

Author: :+1:

> used to communicate each other.

Reviewer: ```suggestion
to communicate with each another.
```

Reviewer: sorry, I meant "with each other"

> Additionally, you could receive statistics data from every endpoint in the network.

Reviewer: ```suggestion
Additionally, one could receive statistical data from every endpoint in the network.
```

Reviewer: impersonal?

This data is very useful to analyze the performance and seek any possible communication problem in the network.

ROS 2 relies on DDS communication protocol to communicate different nodes.
There are different RMWs (ROS MiddleWare) available in the latest ROS 2 versions.

> Just be sure to use Fast DDS as your ROS 2 middleware in order to receive statistic data and monitor your network.

Reviewer: ```suggestion
Make sure you are using Fast DDS as your ROS 2 middleware in order to receive statistical data and monitor your network.
```

> In the following sections are described how to use Fast DDS with Statistics active in different ROS2 versions.

Reviewer: ```suggestion
In the following sections, instructions for using Fast DDS with Statistics in different ROS2 versions are provided, as well as a brief tutorial demonstrating the integration of Fast DDS Monitor with ROS2 in a simple scenario.
```
